utilities/fault_injection_fs.cc

```diff
@@ -501,8 +505,7 @@ IOStatus FaultInjectionTestFS::InjectError(ErrorOperation op,
               // The randomly generated string could be identical to the
               // original one, so retry
               do {
-                str = DBTestBase::RandomString(&ctx->rand,
-                                    static_cast<int>(len));
+                str = ctx->rand.RandomString(static_cast<int>(len));
```

Comment:
I was a little concerned about 'git blame's ability to track this rename with level of change, but I checked out, collapsed to one commit, and the git blame works fine. :)